src/components/header/Header.tsx

```diff
@@ -1,23 +1,15 @@
 import Image from 'next/image';
 import Link from 'next/link';
 import logo from 'public/logo.svg';
-import Head from 'next/head';
-import BannerImage from '@/components/header/BannerImage';
 
 export const Header = () => {
-  return (
-    <header>
-      <Head>
-        <link rel='preload' href='/banners/BreakpointSmall.webp' as='image' />
-        <link rel='preload' href='/banners/BreakpointMedium.webp' as='image' />
-        <link rel='preload' href='/banners/BreakpointLarge.webp' as='image' />
-      </Head>
-      <nav className='sticky top-0 z-10 mx-auto flex bg-white p-4 lg:ps-16'>
-        <Link href='/'>
-          <Image src={logo} alt='Company logo' loading='lazy' />
-        </Link>
-      </nav>
-      <BannerImage />
-    </header>
-  );
+    return (
+        <header className='sticky top-0 z-10'>
+            <nav className='mx-auto flex bg-white p-4 lg:ps-16'>
+                <Link href='/'>
+                    <Image src={logo} alt='Company logo' loading='lazy' />
```

Comment:
🟡 since the logo is above the fold, lazy loading is not the best idea here. I think it would be even better to either give it priority or leave out the loading prop from it alltogether